assisted-events-scrape/tests/test_cluster_events_worker.py

```diff
@@ -0,0 +1,103 @@
+import unittest
```

Comment:
Why unittest? We are using pytest on test-infra and I think it is better to use the same infrastructure for testing on all of our projects.

Reply:
No real reason for unittest, just did not think about checking out test-infra. I agree with you, I will change it to pytest.

Reply:
@eliorerz what package would you recommend for mocking? AFAIU unittest.mock is the one recommended also when using pytest, is this correct?

Reply:
yes, I think ``unittest.mock`` is still the way to go